blazor/dropdown-list/data-binding.md

```diff
@@ -7,68 +7,377 @@ control: DropDown List
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/data-binding/enum-data-binding.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with Enum data binding](./images/data-binding/blazor_dropdown_enum-data-binding.png)
+
+### ValueTuple data binding
+
+You can bind [ValueTuple](https://learn.microsoft.com/en-us/dotnet/api/system.valuetuple-2?view=net-5.0) data to the DropDownList component. The following code helps you to get a string value from the enumeration data by using `ValueTuple`
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/data-binding/value-tuple-data-binding.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with ValueTuple data binding](./images/data-binding/blazor_dropdown_value-tuple-data-binding.png)
+
+## Binding remote data 
+
+The DropDownList loads the data from remote data services through the DataSource property. 
+
+The DropDownList supports retrieval of data from remote data services with the help of [DataManager](https://blazor.syncfusion.com/documentation/data/getting-started)  control. The [Query](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_Query) property is used to fetch data from the database and bind it to the DropDownList.
+
+* [DataManager.Url](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DataManager.html#Syncfusion_Blazor_DataManager_Url) - Defines the service endpoint to fetch data.
+* [DataManager.Adaptor](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DataManager.html#Syncfusion_Blazor_DataManager_Adaptor) - Defines the adaptor option. By default, ODataAdaptor is used for remote binding. The adaptor is responsible for processing responses and requests from/to the service endpoint. 
+* [Syncfusion.Blazor.Data](https://www.nuget.org/packages/Syncfusion.Blazor.Data/) package provides some predefined adaptors that are designed to interact with particular service endpoints.
+
+### OData v4 services
+
+The [OData v4 Adaptor](https://blazor.syncfusion.com/documentation/data/adaptors#odatav4-adaptor) provides ability to consume and manipulate data from OData v4 services. The following sample displays the first six customer details from `Customers` table of the `Northwind` Data Service.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/data-binding/odata-v4-services.razor %}
+
+{% endhighlight %}
+
+![Blazor DropdownList with OData v4 Adaptor](./images/data-binding/blazor_dropdown_odata-v4-services.png)
+
+### Web API adaptor
+
+The [Web Api Adaptor](https://blazor.syncfusion.com/documentation/data/adaptors#web-api-adaptor) use this to interact with Web API created under OData standards. The `WebApiAdaptor` is extended from the [ODataAdaptor](https://blazor.syncfusion.com/documentation/data/adaptors#odata-adaptor). Hence use `WebApiAdaptor`, the endpoint should understand the OData formatted queries sent along with request. 
```

Comment:
The [Web Api Adaptor](https://blazor.syncfusion.com/documentation/data/adaptors#web-api-adaptor) use this to interact with Web API created under OData standards. The `WebApiAdaptor` is extended from the [ODataAdaptor](https://blazor.syncfusion.com/documentation/data/adaptors#odata-adaptor), hence use the `WebApiAdaptor.` The endpoint should understand the OData formatted queries sent along with the request.